modules/charon-core/src/main/java/org/wso2/charon3/core/schema/SCIMConstants.java

```diff
@@ -27,7 +27,8 @@ public class SCIMConstants {
     public static final String USER_CORE_SCHEMA_URI = "urn:ietf:params:scim:schemas:core:2.0:User";
     public static final String ENTERPRISE_USER_SCHEMA_URI =
             "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User";
-    public static final String CUSTOM_USER_SCHEMA_URI = "urn:scim:wso2:schema";
+    public static final String SYSTEM_USER_SCHEMA_URI = "urn:scim:wso2:schema";
+    public static final String CUSTOM_USER_SCHEMA_URI = "urn:scim:schemas:extension:custom:User";
```

Comment:
Introduce a new variable for custom schema URI and keep the old custom schema URI var as it is.

Reply:
Make the old var deprecated

Reply:
Fixed with 77c39056d8f8af024c736ceb245bc72714ac49ac